lstn/static/js/controllers.js

```diff
@@ -379,16 +407,21 @@ angular.module('lstn.controllers', [])
       if (!data || !data.key) {
         $scope.isCurrentController = false;
         $scope.playing = null;
-        apiswf.rdio_stop();
+
+        if (apiswf) {
```

Comment:
Obviously we don't want to do anything if `apiswf` is undefined/null, but do we want to error or provide some user feedback if they try to do something but the swf file doesn't exist?